### PR TITLE
chore: update deps(synth-agent, pw)

### PR DESCRIPTION
**Pull request by kyungeunni**
## Summary
- Updates deps: `@elastic/synthetics@beta.36` and `playwright@1.26.0`
- Fix README.md PW update instruction
- Updates code generation code to catch up [upstream changes](https://github.com/microsoft/playwright/blob/main/packages/playwright-core/src/server/recorder/javascript.ts#L40)

### CODE REVIEW

File: electron/execution.ts [hunk 1]
```diff
@@ -137,7 +137,7 @@ function onRecordJourneys(mainWindowEmitter: EventEmitter) {
       await (context as any)._enableRecorder({
         launchOptions: {},
         contextOptions: {},
-        startRecording: true,
+        mode: 'recording',
```
Comment (kyungeunni) [inline electron/execution.ts:140]: Changes due to a PW change

File: electron/syntheticsGenerator.ts [hunk 2]
```diff
@@ -212,27 +223,25 @@ export class SyntheticsGenerator extends PlaywrightGenerator.JavaScriptLanguageG
...
       if (emitPromiseAll) {
         formatter.add(']);');
-      } else if (signals.assertNavigation) {
-        formatter.add(
-          `  expect(${pageAlias}.url()).toBe(${quote(signals.assertNavigation.url ?? '')});`
-        );
+      } else if (signals.assertNavigation?.url) {
+        // original code uses the full URL including query strings which often contains randomly generated value(cashbusting)
+        // when replaying the script it will fail so we will check only baseurl and path
+        const url = new URL(signals.assertNavigation.url);
```
Comment (vigneshshanmugam) [inline electron/syntheticsGenerator.ts:243]: Does it parse query by default? can you disable if it does?

**Review by vigneshshanmugam — APPROVED** [review]
LGTM

File: electron/syntheticsGenerator.ts (hunk 2 above)
Comment (kyungeunni) [reply electron/syntheticsGenerator.ts:243]: It does parse query by default - and it doesn't seem to have an option for disabling it. Can I ask what's your concern about query?

**Pull request merged**